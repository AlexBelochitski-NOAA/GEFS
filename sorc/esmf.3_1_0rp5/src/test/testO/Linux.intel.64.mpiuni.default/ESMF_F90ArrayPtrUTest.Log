20190610 112230.065939 INFO     PET0 Running with ESMF Version 3.1.0rp5
20190610 112230.066213 INFO     PET0  Beginning Test, file ESMF_F90ArrayPtrUTest.F90, line         249
20190610 112230.066231 INFO     PET0  NUMBER_OF_PROCESSORS           1
20190610 112230.066269 INFO     PET0  PASS  1D Integer array pointer size test, ESMF_F90ArrayPtrUTest.F90, line         260
20190610 112230.066291 INFO     PET0  PASS  2D Integer array pointer size test, ESMF_F90ArrayPtrUTest.F90, line         271
20190610 112230.066303 INFO     PET0  PASS  3D Integer array pointer size test, ESMF_F90ArrayPtrUTest.F90, line         282
20190610 112230.068618 INFO     PET0  PASS  4D Integer array pointer size test, ESMF_F90ArrayPtrUTest.F90, line         293
20190610 112230.068631 INFO     PET0  PASS  5D Integer array pointer size test, ESMF_F90ArrayPtrUTest.F90, line         304
20190610 112230.068641 INFO     PET0  PASS  6D Integer array pointer size test, ESMF_F90ArrayPtrUTest.F90, line         315
20190610 112230.068651 INFO     PET0  PASS  7D Integer array pointer size test, ESMF_F90ArrayPtrUTest.F90, line         326
20190610 112230.068735 INFO     PET0  Number of failed tests:           0
20190610 112230.068742 INFO     PET0  Ending Test, file ESMF_F90ArrayPtrUTest.F90, line         578
