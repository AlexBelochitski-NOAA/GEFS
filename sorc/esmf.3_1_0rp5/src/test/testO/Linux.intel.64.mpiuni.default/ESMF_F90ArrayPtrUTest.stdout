  Beginning Test, file ESMF_F90ArrayPtrUTest.F90, line         249
  NUMBER_OF_PROCESSORS           1
  PASS  1D Integer array pointer size test, ESMF_F90ArrayPtrUTest.F90, line     
     260
  PASS  2D Integer array pointer size test, ESMF_F90ArrayPtrUTest.F90, line     
     271
  PASS  3D Integer array pointer size test, ESMF_F90ArrayPtrUTest.F90, line     
     282
  PASS  4D Integer array pointer size test, ESMF_F90ArrayPtrUTest.F90, line     
     293
  PASS  5D Integer array pointer size test, ESMF_F90ArrayPtrUTest.F90, line     
     304
  PASS  6D Integer array pointer size test, ESMF_F90ArrayPtrUTest.F90, line     
     315
  PASS  7D Integer array pointer size test, ESMF_F90ArrayPtrUTest.F90, line     
     326
  Number of failed tests:           0
  Ending Test, file ESMF_F90ArrayPtrUTest.F90, line         578
Match: rank 1 Fortran 90 pointer is 72 bytes, matches computed size ok
Match: rank 2 Fortran 90 pointer is 96 bytes, matches computed size ok
Match: rank 3 Fortran 90 pointer is 120 bytes, matches computed size ok
Match: rank 4 Fortran 90 pointer is 144 bytes, matches computed size ok
Match: rank 5 Fortran 90 pointer is 168 bytes, matches computed size ok
Match: rank 6 Fortran 90 pointer is 192 bytes, matches computed size ok
Match: rank 7 Fortran 90 pointer is 216 bytes, matches computed size ok
